### PR TITLE
docs: Cross Program Linelisting Dimensions endpoint [DHIS2-13411]

### PR DESCRIPTION
**Pull request by gnespolino**

### CODE REVIEW

File: src/developer/web-api/analytics.md [hunk 1]
```diff
@@ -2128,6 +2129,20 @@ Data elements and tracked entity attributes are considered *supported types* if
 - `BOOLEAN`
 - `TRUE_ONLY`
 
+### Tracked Entities analytics dimensions
+
+#### Tracked Entities query analytics dimensions { #webapi_teis_query_analytics_dimensions }
+
+The `/analytics/teis/query/dimensions?trackedEntityType=TET` resource accepts a mandatory id of a tracked entity type `TET` and returns the following data dimensions:
+
+for each program `P` such that a Tracked Entity Instance of type `TET` can register in program `P` :
```
Comment (maikelarabori) [inline src/developer/web-api/analytics.md]: Should we use all lowercase as we do for program, data element, etc.?
`Tracked Entity Instance` -> `tracked entity instance`

File: src/developer/web-api/analytics.md [hunk 2]
```diff
@@ -2128,6 +2129,20 @@ Data elements and tracked entity attributes are considered *supported types* if
...
+- **Program indicators** connected to `P`
+- **Data elements** of *supported types* in `P`, with program stage for each data element
+- **Tracked entity attributes** of *supported types* associated with the program that are not confidential
+- **Program Attribute** of `P`
```
Comment (maikelarabori) [inline src/developer/web-api/analytics.md]:
```suggestion
- **Program attributes** of `P`
```

File: src/developer/web-api/analytics.md [hunk 3]
```diff
@@ -2128,6 +2129,20 @@ Data elements and tracked entity attributes are considered *supported types* if
...
 - `TRUE_ONLY`
 
+### Tracked Entities analytics dimensions
+
+#### Tracked Entities query analytics dimensions { #webapi_teis_query_analytics_dimensions }
+
+The `/analytics/teis/query/dimensions?trackedEntityType=TET` resource accepts a mandatory id of a tracked entity type `TET` and returns the following data dimensions:
+
+for each program `P` such that a Tracked Entity Instance of type `TET` can register in program `P` :
+- **Program indicators** connected to `P`
```
Comment (maikelarabori) [inline src/developer/web-api/analytics.md]:
```suggestion
- **Program indicators** associated to `P`
```

File: src/developer/web-api/analytics.md (hunk 1 above)
Comment (maikelarabori) [inline src/developer/web-api/analytics.md]:
```suggestion
for each program `P` associated with a tracked entity instance of type `TET`:
```

**Review by maikelarabori — APPROVED** [review]
Looks good, just a few minor comments @gnespolino :+1:

**Review by d-bernat — APPROVED** [review]

**Review by Philip-Larsen-Donnelly — APPROVED** [review]

**Pull request merged**